gpu-service/jobs.py

```
@@ -385,6 +438,18 @@ def process_audio_upload(bucket_name, object_name, event_data=None):
             f.write(text)
         
+        # Append transcript to DeepLake text dataset for downstream RAG
+        try:
+            transcripts_ds.append({
+                "text": [text],
+                "bucket": [bucket_name],
+                "object_name": [object_name],
+                "source_video": [event_data.get("source_video") if event_data else None],
+            })
+            transcripts_ds.commit()
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)